mobile/lib/src/presentation/navigation/routes/router.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:auto_route/auto_route_annotations.dart';
+import 'package:polydodo/src/presentation/navigation/navdrawer_widget.dart';
+import 'package:polydodo/src/presentation/pages/bluetooth_page/bluetoothSelector_page.dart';
+import 'package:polydodo/src/presentation/pages/dashboard/dashboard_page.dart';
+import 'package:polydodo/src/presentation/pages/record_sleep/record_sleep_guide_page.dart';
+
+@MaterialAutoRouter(
+    generateNavigationHelperExtension: true,
+    routes: <AutoRoute>[
+      MaterialRoute(page: NavDrawerPage),
+      MaterialRoute(page: DashBoardPage, initial: true),
+      MaterialRoute(page: RecordSleepGuidePage),
+      MaterialRoute(page: RecordSleepValidatePage),
+      MaterialRoute(page: BluetoothSelectorPage),
```

Comment:
Why did we renamed Route for Page?

Reply:
I tought it sounded more smooth to have "routes that point to pages which loads widgets". But I might be completely off.